### ISSUE TITLE
Can you offer the train.log 

### ISSUE THREAD

**Issue by ZhangLei999** [jfzhang95/PoseAug#24]
In a function called project_to_2d, I can't understand the code :
tan = torch.sum(p * XX, dim=len(XX.shape) - 1, keepdim=True).

**Comment by ZhangLei999** [comment]
tan = p1x+p2y
XX * tan + p*r2 = (x, y) * (p1x+p2y) + p1r^2 +p2r^2 = p1x^2 + p2xy + p1xy + p2y^2+ p1r^2 +p2r^2

It doesn't match the following formula:
x_d = x + [2p1xy + p2(r^2 + 2x^2)]
y_d = y + [2p2xy + p1[r^2 + 2y^2]]

[2p1xy + p2(r^2 + 2x^2)] + [2p2xy + p1[r^2 + 2y^2]]  != XX * tan + p*r2

**Comment by Garfield-kh** [comment]
Hi, thank you for the interest.
Here is the train [log](https://drive.google.com/file/d/16QMwhWwavd8UOPqc9lKZVfddM-Aa8Bt6/view?usp=sharing).
